lib/tether/toolbox.go

```diff
@@ -137,6 +148,11 @@ func (t *Toolbox) containerAuthenticate(_ toolbox.VixCommandRequestHeader, data
 	if err := c.UnmarshalBinary(data); err != nil {
 		return err
 	}
+
+	if t.config == nil {
```

Comment:
I'm not sure we should be returning an error here at all - it's a safety check that we're operating on the VM we think we are rather than anything else. vSphere takes care of the are we authenticated question.

If we are returning an error I think "not yet initialized" is a better one - it implies that a second attempt may succeed.